src/main/java/org/aksw/simba/lemming/creation/Inferer.java

```diff
@@ -41,9 +42,10 @@
 public class Inferer {
 
 	private static final Logger LOGGER = LoggerFactory.getLogger(Inferer.class);
-
-	public Inferer() {
-
+	private boolean isMat = false;

```

Comment:
What is the meaning of this flag? It seems like it is not documented anywhere :wink: